analyses/doublet-detection/run_doublet-detection.sh

```diff
@@ -8,15 +8,19 @@ set -euo pipefail
 MODULE_DIR=$(dirname "${BASH_SOURCE[0]}")
 cd ${MODULE_DIR}
 
+NB_DIR=exploratory-analyses # directory with exploratory notebooks
```

Comment:
I think it would make more sense to store this in `teplate-notebooks`. I feel like exploratory analysis is something that you are running without parameters and we want to commit both the Rmd file and the rendered notebook. For this, you have just created a template so I would move that there.